_data/projects/OpinionatedGuides.yml

```
@@ -0,0 +1,11 @@
+name: Opinionated Guides
+desc: a one-stop guide to computers, linux, and electronics.
+site: https://github.com/opinionatedguide/opinionatedguide.github.io-src
+tags:
+- awesome
+- documentation
+- python
+- bash
+upforgrabs:
+  name: hacktoberfest
+  link: https://github.com/opinionatedguide/opinionatedguide.github.io-src/issues/2
```

This link should go to the list of available issues, not just to a specific . issue

```suggestion
  link: https://github.com/opinionatedguide/opinionatedguide.github.io-src/labels/Hacktoberfest
```